x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.ts

```diff
@@ -97,11 +99,28 @@ export const buildEventsSearchQuery = ({
     { bool: { filter: [{ bool: { should: [...rangeFilter], minimum_should_match: 1 } }] } },
   ];
 
+  const sort = [];
```

Comment:
We have an implicit return type on this function, and a [cast on the consumer](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/server/lib/detection_engine/signals/single_search_after.ts#L75) to satisfy typescript; it would be great to remove that cast and make the return value explicit here.

Failing that, typing this `sort` variable would at least be a step in the right direction.

Reply:
typed as `sort: estypes.SearchSort`